lib/api/include/irods/get_resource_info_for_operation.h

```diff
@@ -0,0 +1,38 @@
+#ifndef IRODS_GET_RESOURCE_INFO_FOR_OPERATION_H
+#define IRODS_GET_RESOURCE_INFO_FOR_OPERATION_H
+/// \file
+
+#include "irods/dataObjInpOut.h"
+
+struct RcComm;
+
+/// Get preferred-resource information for the specified operation.
+///
+/// \parblock
+/// \param[in] _conn - An rcComm_t connection handle to the server.
```

Comment:
Remove hyphen following all parameter names.

For example:
```cpp
/// \param[in] _conn An RcComm ...
```

This only applies to lines containing `\param[*]`.